.eslintrc.json

```diff
@@ -0,0 +1,16 @@
+{
+	"env": {
+		"es6": true,
+		"jest": true,
```

Comment:
Will push up ESLint for Jest in the morning, I ran out of time today because _real life_ got in the way of coding 😝